### ISSUE TITLE
InnerBlocks area disappears after changing the block alignment

### ISSUE THREAD

**Issue by CreativeDive** [AdvancedCustomFields/acf#307]
Hey @elliotcondon,

today I've found a new issue while working with innerBlocks.

If I add new ACF block with an innerBlocks area, the inserter area is disappears immediately after changing the block alignment (full, wide, center, ...).

So we can't add innerBlocks for this section after changing the block alignment. This happens only after add a new block, if the first change is the block alignment.

![innerblocks-bug](https://user-images.githubusercontent.com/31563167/83241266-dba0b780-a19a-11ea-8343-7ced94309b44.gif)

Tested with latest ACF beta 5.9 + Gutenberg 8.2.1

**Comment by elliotcondon** [comment]
Hi @CreativeDive 

Thanks for the bug report. I can confirm the issue exists only when the Gutenberg plugin (8.2.1) is active. It's hard to know if this kind of problem is due to a change in Guten-code, or a conflict between WordPress core editor and the Guten-plugin.

**Comment by elliotcondon** [comment]
@CreativeDive I haven't had much luck diagnosing the exact cause of this issue. I'll be sure to keep this issue in mind, but suspect it will resolve itself with an upcoming Gutenberg plugin release.

**Comment by elliotcondon** [comment]
@CreativeDive Just checking in on this issue. In recent testing, I haven't been able to replicate the bug - perhaps it has been fixed within the Gutenberg plugin? Can you confirm the same?

**Comment by CreativeDive** [comment]
@elliotcondon It seems this was fixed with a gutenberg update in the past. It works fine now.
